client/components/IntroModule/UserForm.tsx

```diff
@@ -101,31 +102,30 @@ export const UserForm = () => {
   return (
     <>
       {!providerInstalled ? (
-        <Box>
+        <Box textColor="LightPurple">
           <SpanText centerText>A Wallet provider is required to use Chat Wallette</SpanText>
         </Box>
       ) : (
-        <Stack row itemSpace="xs" shrink padding>
+        <Stack row itemSpace="xs" padding="xs" shrink>
           {authData !== null &&
           walletStatus === WalletStatus.Unlocked &&
           authenticatingStatus === AuthenticatingStatus.Unauthenticated ? (
-            <Stack row itemSpace="xs" shrink padding>
+            <Stack row itemSpace="xs" shrink>
               <InputField
+                type="text"
                 value={screenName ?? ""}
                 onChange={onChange}
                 onKeyDown={onKeyDown}
                 size={16}
                 autoFocus
-                placeholder="Select a name"
+                placeholder="Enter username"
```

Comment:
I'm really uncomfortable with Enter username, the implementation is they already have one. Let's go with Choose username